samples/Azure.Network.Management.Interface/Generated/Internal/ModelSerializationExtensions.cs

```diff
@@ -245,7 +245,7 @@ public static void WriteObjectValue(this Utf8JsonWriter writer, object value)
             }
         }
 
-        private static class TypeFormatters
+        internal static class TypeFormatters
```

Comment:
It looks like this may apply the change to Azure libraries - the new ClientUriBuilder should be used for unbranded only for now until we have time to do the validation that changing the solution in Azure clients won't cause perf regressions.

Reply:
typeformatters is shared between them so we need to mark it as internal for all